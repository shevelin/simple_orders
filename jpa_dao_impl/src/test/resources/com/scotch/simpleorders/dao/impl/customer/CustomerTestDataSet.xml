<!--
<dataset>
&lt;!&ndash;
    <customer id="nextval('customer_id_seq')" name="Serious Sam1" password="password"/>
    <customer id="nextval('customer_id_seq')" name="Wild Bill1" password="password"/>
    <customer id="nextval('customer_id_seq')" name="Dieter Bohlen1" password="password"/>
    <customer id="nextval('customer_id_seq')" name="Tom Cruise1" password="password"/>
    <customer id="nextval('customer_id_seq')" name="Vasya1" password="password"/>
&ndash;&gt;
</dataset>

-->
<!--<!DOCTYPE dataset SYSTEM "dataset.dtd">-->
<dataset>
    <table name="customer">
        <column>id</column>
        <column>name</column>
        <column>password</column>
<!--
        <row>
            <value>1</value>
            <value>Serious Sam1</value>
            <value>password</value>
        </row>
        <row>
            <value>2</value>
            <value>Wild Bill1</value>
            <value>password</value>
        </row>
-->
    </table>
</dataset>