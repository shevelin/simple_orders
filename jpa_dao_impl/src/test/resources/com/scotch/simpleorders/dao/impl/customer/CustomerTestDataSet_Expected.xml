<!--
<dataset>
    <customer id="1" name="Serious Sam" password="password"/>
</dataset>
-->

<dataset>
<table name="customer">
    <column>id</column>
    <column>name</column>
    <column>password</column>
    <row>
        <value>1</value>
        <value>Serious Sam</value>
        <value>password</value>
    </row>
</table>
</dataset>